CONTRIBUTING.md

```diff
@@ -0,0 +1,56 @@
+# Contributing
+
+Thank you for considering a contribution to FIDO!
+This document outlines the change submission process for FIDO, along with our standards for new code contributions.
+Following these guidelines helps us assess your changes faster and makes it easier for us to merge you submission!
+
+Contributions can be many things, including submitting bug reports, writing documentation or writing code which can be incorporated into FIDO.
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+- [Submitting bugs](#submitting-bugs)
+- [Submitting code changes](#submitting-code-changes)
+- [Code Review & Approval](#code-review-&-approval)
+- [Release process](#release-process)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Submitting bugs
+
+FIDO uses Github Issues to track bugs.
+If you find a problem, please check existing bugs before submitting your issue.
+
+Useful questions to answer if you're having problems include:
+
+* What version of FIDO are you using?
+* How was FIDO installed? Via pip, manually using a git checkout, etc.
+* What did you do to cause this bug to happen?
+* What did you expect to happen?
+* What did you see instead?
+* Can you reproduce this reliably?
+
+
+## Submitting code changes
+
+Every new feature and bugfix is expected to go through code review before inclusion.
+We use Github pull requests for code review.
+
```

Comment:
Possibly PEP 257 for docstrings.  We might also want to discuss: Python 2 and/or 3 support, line length, semantic line breaks, requiring tests, commit message style.